README.md

```diff
@@ -1,2 +1,20 @@
-# hello_world
-Project Hello World week 1
+Project Name
```

Comment:
- [ ] [Optional] The branch should have the name of a feature or action e.g. ```'created_sidebar'``` or ```'added_comments_section'```, please consider doing this in future projects, you can skip this for now

Reply:
Does that mean that I shouldn't name the branch as "feature", but instead I should use a more descriptive name?